ci/nightly/pipeline.yml

```diff
@@ -674,6 +675,17 @@ steps:
           composition: output-consistency
           args: ["--seed=$BUILDKITE_JOB_ID", "--max-runtime-in-sec=1200"]
 
+  - id: postgres-consistency-test
+    label: "Postgres consistency test"
```

Comment:
You can consider renaming this to "Output consistency test (Postgres)" or somehow indicate that this and the previous Buildkite step belong to the same family.

It may also be a good idea to have the two ids start with the same prefix, that is, name this step `output-consistency-test-postgres` or something along those lines.

Reply:
Will do

Reply:
Changed the Buildkite ids and names in https://github.com/MaterializeInc/materialize/pull/21781/commits/049582dfbe1b751902e7703d47af98dc9db81057.
Let me know if you would like me to rename the folders as well.